lenovo/thinkpad/z/gen2/z13/default.nix

```diff
@@ -7,7 +7,7 @@
 
   environment.etc."asound.conf".source = ./asound.conf;
 
-  networking.networkmanager.fccUnlockScripts = [
+  networking.modemmanager.fccUnlockScripts = [
```

Comment:
I think we need to set this depending on the NixOS version here.

You can use a conditional like

```
(lib.versionOlder lib.version "25.05pre")
```

to include the right code for both 24.11 and 25.05.